apps/files/src/views/favorites.ts

```diff
@@ -106,6 +106,21 @@ export default () => {
 		removePathFromFavorites(node.path)
 	})
 
+	/**
+	 * Update favourites navigation when a folder is renamed
+	 */
+	subscribe('files:node:renamed', (node: Node) => {
+		if (node.type !== FileType.Folder) {
+			return
+		}
+
+		if (node.attributes.favorite !== 1) {
```

Comment:
Maybe to make sure if we ever fix the type to be a real boolean

```suggestion
		if (!!node.attributes.favorite) {
```

Reply:
I prefer explicit comparisons, I have seen too many issues without, especially with lightly typed languages like typescript. :)